kairon/shared/account/processor.py

```diff
@@ -79,31 +81,30 @@ def check_bot_exists(name: str, account: int, raise_exception: bool = True):
         return bot_exists
 
     @staticmethod
-    async def add_bot_with_template(name: str, account: int, user: str, is_new_account: bool = False,
-                                    add_default_data: bool = True, template_name: str = None, **metadata):
+    async def add_bot_with_template(name: str, account: int, user: str, template_name: str = None):
         """
         add a bot to account and apply template
 
-        :param metadata: metadata of new bot
         :param name: bot name
         :param account: account id
         :param user: user id
-        :param is_new_account: True if it is a new account
-        :param add_default_data: True if default data is to be added
         :param template_name: template name
         :return: bot id
         """
         from kairon.shared.data.processor import MongoProcessor
 
-        bot = AccountProcessor.add_bot(name, account, user, is_new_account=is_new_account,
-                                       add_default_data=add_default_data, **metadata)
+        metadata = {"metadata": {"from_template": template_name}}
+        add_default_data = True if Utility.check_empty_string(template_name) else False
+        bot = AccountProcessor.add_bot(name, account, user, False, add_default_data, **metadata)
         bot_id = bot['_id'].__str__()
         if not Utility.check_empty_string(template_name):
-            input_path = "./template/use-cases/Hi-Hello-GPT/models/20230730-084056.tar.gz"
+            input_path = Utility.get_latest_file(f"template/use-cases/{template_name}/models", "20230730-084056.tar.gz")
```

Comment:
remove "20230730-084056.tar.gz"
why is it again hard coded after we removed it y'day?